Dockerfile

```diff
@@ -31,15 +31,14 @@ ENV CGO_ENABLED=0 \
     GOPROXY=${GOPROXY} \
     GOOS=${TARGETOS} \
     GOARCH=${TARGETARCH} \
-    GOARM=${TARGETVARIANT} \
+    GOARM=7 \
```

Comment:
Is this change necessary?

What about arm32v6 arm32v7 arm64v8?

Reply:
Agree with @kaovilai , let's make it configurable

Reply:
updated